src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -560,6 +560,7 @@
     <Field Name="ExplicitCastInCode" Type="Boolean"/>
     
     <Field Name="ConstantValueOpt" Type="ConstantValue" Null="allow"/>
+    <Field Name="IsNullable" Type="bool?" Null="allow"/>
   </Node>
```

Comment:
This is going to be confusing. There is a category of conversion in the existing C# language specification called nullable conversions. I think this is not intended to have that meaning. If that is true, we need a better name for it.

Reply:
Renamed to `IsExplicitlyNullable`.

---
In reply to: [175559725](https://github.com/dotnet/roslyn/pull/25516#discussion_r175559725) [](ancestors = 175559725)